plugins/chat/assets/stylesheets/desktop/chat-channel.scss

```diff
@@ -0,0 +1,16 @@
+@media screen and (max-width: 1000px) {
```

Comment:
Dont we have a predefined breakpoint instead of hardcoding px?

Reply:
I don't know if we have this exact case, I just decided to follow what slack does as they have the exact same case. Happy to use a different variable if you have a suggestion.

Reply:
sure thanks